tools/clang/test/DXC/strip_dbg.test

```diff
@@ -0,0 +1,18 @@
+// /Fd implies /Qstrip_debug ; path with \ produces auto hash-named .pdb file
+// RUN: %dxc /T ps_6_0 %S/Inputs/smoke.hlsl /Zi /Fd ./ /Fo %t.strip
```

Comment:
can we include the check for the generated .pdb file? Should we delete it afterwards since filecheck isn't' aware of its generation?

Reply:
It was checked by *.pdb in hcttestcmds.cmd.
I'm not sure it is OK to run 
  rm *.pdb in lit or not.

Reply:
changed to cat *.pdb | rm *.pdb
linux passed.
Hope this will work.

Reply:
In conversation with Chris, I no longer think deleting these files explicitly is necessary. I misunderstood how lit identifies which files need deletion